### ISSUE TITLE
Flexible Content Categorization System

### ISSUE THREAD

**Issue by Chikedor** [RSSNext/Follow#598]
### Clear and concise description of the problem

As a user of this project, I want a flexible content categorization system that includes Articles, Social Media, Pictures, Videos, Audio, and Notifications, so that I can efficiently organize and filter my content based on my preferences.

### Suggested solution

In the app's content organization system, we could:

1. Implement preset content types: Articles, Social Media, Pictures, Videos, Audio, and Notifications
2. Allow users to add, remove, or customize these content types
3. Develop a filtering system that enables users to show/hide specific content types
4. Use clear terms like "labels" or "tags" to avoid confusion

### Alternative

An alternative could be a tag-based system where users can create and assign multiple tags to each feed, allowing for more granular and flexible organization.

### Additional context

The current organizational structure is confusing. This new system would help users focus on the content types they're most interested in, improving overall user experience and engagement. Similar systems exist in email clients (labels) and social media platforms (hashtags).

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by kiFte** [comment]
Right-click on the subscription feed, Edit, which has the option to add custom categories. 
Articles, social media, images, videos, audio and notifications may be views, not categories.

**Comment by kiFte** [comment]
The bigger problem is that adjusting custom categories is now too much of a hassle.
